doc/rados/operations/control.rst

```diff
@@ -214,8 +218,8 @@ run the following command:
 
    ceph osd crush move {id} {loc1} [{loc2} ...]
 
-To set the CRUSH weight of a specific OSD (specified by ``{name}``) to
-``{weight}``, run the following command:
+To set the weight of a specific OSD (specified by ``{name}``) to ``{weight}``,
```

Comment:
And here, to disambiguate vs the override reweight

Reply:
It has also been added back in.